docs/reference/publication_formats.md

```diff
@@ -1,12 +1,152 @@
+
 ## JSON
 
+This section describes how to publish data in JSON format.
+
+If your data is small enough to fit into memory or if you are publishing data via an API, you should use the [small files and API responses option](#small-files-and-api-responses-option). If your data is too large to fit into memory, you should use the [streaming option](#streaming-option).
+
+For guidance on paginating or streaming individual networks, see [how to publish large networks](../guidance/publication.md#how-to-publish-large-networks).
+
+### Small files and API responses option
+
+The network package schema describes the structure of the container for publishing one or more networks in JSON format and for supporting pagination.
+
+For this version of OFDS, the canonical URL of the schema is [https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json](https://raw.githubusercontent.com/Open-Telecoms-Data/open-fibre-data-standard/0.1.0-alpha/schema/network-package-schema.json). Use the canonical URL to make sure that your software, documentation or other resources refer to the specific version of the schema with which they were tested.
+
+This page presents the schema in an interactive browser, you can also download the canonical version of the schema as [JSON Schema](../../schema/network-package-schema.json).
+
+A network package is a JSON object that must include `.networks`, an array of `Network` objects as described by the [network schema](schema.md). For data published via a paginated API, the optional `.pages` object should be used to provide links to the next and previous pages of results.
+
+::::{tab-set}
+
+:::{tab-item} Schema browser
+Click on schema elements to expand the tree, or use the '+' icon to expand all elements. Use { } to view the underlying schema for any section. Required fields are indicated in **bold**.
+
+ <script src="../_static/docson/widget.js" data-schema="../network-schema.json"></script> 
+:::
+
+:::{tab-item} Small file example
+The following example shows a network package containing two networks:
+```{jsoninclude} ../../examples/json/network-package.json
+:jsonpointer:
+```
+:::
+
+:::{tab-item} API response example
+The following example shows a network package containing two networks with links to the next and previous pages of results.
+```{jsoninclude} ../../examples/json/api-response.json
+:jsonpointer:
+```
+:::
+
+::::
+
+### Streaming option
+
+The streaming option describes how to package multiple JSON-format networks with support for streaming. You should only use this option if your data is too large to load into memory.
+
+The streaming option is a [JSON Lines](https://jsonlines.org/) file in which each line is a valid OFDS network, as described by the [network schema](schema.md).
+
+The following example shows a JSON Lines file containing two networks:
+
+```
+{"id": "1"}
```

Comment:
Again I think a few more example fields would be useful here

Reply:
Done in 2b4ae25476f61df5925ea849a54a3597d64ad6ab